flocker/cli/functional/test_sshconfig.py

```diff
@@ -64,7 +64,13 @@ def test_connection_failed(self):
 
         exc = self.assertRaises(CalledProcessError,
                                 self.configure_ssh, b"127.0.0.1", port)
-        self.assertIn(b"refused", exc.output)
+        # There are different error messages on different platforms.
```

Comment:
It'd be nice if there was some comment on *why* there are different error messages.